example/modules/example.ts

```diff
@@ -160,9 +160,9 @@ thin=${res.thin}`
         } catch (error) {
             msg.channel.send(
                 "error! " +
-                    (error || "")
+                    (error as Error || "")
```

Comment:
The LSP tells me `error` is `any` before the cast which makes sense because this is all valid, albeit bad (no stack accessible) JS:
```js
> throw 1;
Uncaught 1
> throw "hi";
Uncaught 'hi'
> throw null
Uncaught null
> throw undefined
Uncaught undefined
```

Reply:
yes, I was hoping a discussion over that. My LSP tell me it's unknown and it is unknown.

![image](https://user-images.githubusercontent.com/43092101/141688245-22000298-f563-4f90-9fcb-d9f32229d133.png)